packages/SwingSet/src/lib/kmarshal.js

```diff
@@ -71,6 +86,10 @@ const kmarshal = makeMarshal(krefOf, kslot, {
 
 export const kser = value => kmarshal.serialize(harden(value));
 
+/**
+ * @param {import('@endo/marshal').CapData<any>} serializedValue
```

Comment:
I think kernel slots are strings...

```suggestion
 * @param {import('@endo/marshal').CapData<string>} serializedValue
```